.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,57 @@
+// /*
+//  * Copyright contributors to the Galasa project
+//  *
+//  * SPDX-License-Identifier: EPL-2.0
+//  */
```

Comment:
Very minor - double comments here, this could just be:

```
/*
 * Copyright contributors to the Galasa project
 *
 * SPDX-License-Identifier: EPL-2.0
 */
```